packages/core/src/bases/chain-wallet.ts

```diff
@@ -388,6 +396,49 @@ export class ChainWalletBase extends WalletBase {
     return await client.sign(this.address, messages, usedFee, memo || '');
   };
 
+  signAA = async (
+    messages: EncodeObject[],
```

Comment:
My JS is lacking, I am not actually sure how this function gets called, or how I can call the implementation specific functions from it